### DISCUSSION TITLE
Can the log in page be disabled?

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#8033] [Q&A]
I'd trying to see if there is a method to disable the log in page, the reason is that I have other authentication method and don't want to manage individual account username and password

**Comment by SchrodingersGat** [comment]
If you have another authentication method successfully configured, the login page should be bypassed already, right?

What other auth method do you have setup already?

**Reply by jesseyu1984918** [reply]
Azure AD

**Reply by matmair** [reply]
If you are using SSO via SAML or OIDC the login is kept enabled as we need a user interaction in some AAD CDA setups.

**Reply by jesseyu1984918** [reply]
So inventree can work with SAML, is this correct?

**Reply by matmair** [reply]
If configured correctly; SAML is complex and only recommended for experts. It also requires HTTPS on nearly all providers
